docs/src/tutorials/smoothness_priors.jl

```diff
@@ -32,9 +32,9 @@ A, Y, W = ACEfit.assemble(data, model.basis);
 
 # In the following we demonstrate the usage of algebraic and gaussian priors. The choices for `σl, σn` made here may seem "magical", but there is a good justification and we plan to automate this in future releases. 
 
-Pa2 = algebraic_smoothness_prior(model.basis; p=2)
-Pa4 = algebraic_smoothness_prior(model.basis; p=4)
-Pg  = gaussian_smoothness_prior( model.basis, σl = (2/r_nn)^2, σn = (0.5/r_nn)^2);
+Pa2 = ACEpotentials.algebraic_smoothness_prior(model.basis; p=2)
```

Comment:
I think this is wrong. It should be `ACE1x.algebratic_smoothness_prior` etc .... But we are almost ready to get rid of ACE1 and ACE1x entirely so I'm not sure it's worth working on this right now.

Reply:
Not sure, I chose `ACEpotentials` because of this https://github.com/ACEsuit/ACEpotentials.jl/blob/4a09741259e370b97a83f18bacc47877a6d0eae2/src/models/smoothness_priors.jl#L90

I don't mind changing to `ACE1x` if that's better, but at minimum the change I made allows that part of the docs to build successfully, where it failed before.

I don't care about this example specifically, but I want to fix the unimportant CI failures so that more impactful things don't get lost.